lms/djangoapps/verify_student/api.py

```diff
@@ -120,6 +136,31 @@ def update_verification_attempt(
             )
             raise VerificationAttemptInvalidStatus
 
+        if status == VerificationAttemptStatus.PENDING:
```

Comment:
I'd recommend emitting the event _after_ the save has completed. If the save fails, we wouldn't want erroneous events to be emitted.